app/models/project.rb

```diff
@@ -415,6 +423,18 @@ def subscribed_repos(user)
     subscriptions.with_repository_subscription.where('repository_subscriptions.user_id = ?', user.id).map(&:repository).uniq
   end
 
+  def dependent_repos_view_query(limit, offset=0)
```

Comment:
Do we need limit and offset to be args at this point?

Reply:
For this commit, strictly speaking, we do not. I was playing around with other sections on my local and for other parts will need it. I can take it out and at it as a part of another PR later if that's preferred.